tensorflow/core/protobuf/config.proto

```diff
@@ -99,6 +99,22 @@ message GPUOptions {
   // the overall host system performance.
   bool force_gpu_compatible = 8;
 
+  // Whether to merge streams in one stream group. Four types of streams will be
+  // created within a stream group by default: one compute stream, one HtoD copy
+  // stream, one DtoH copy stream, and several DtoD copy streams. Use the
+  // STREAM_MERGE_OPTIONS to merge the copy streams into the compute stream. For
+  // example, setting "merge_h_to_d_stream = true" to make the compute stream
+  // responsible for both computation and HtoD memory copy. Stream merging helps
+  // reduce the overhead caused by stream synchronization, especially when data
+  // transfers are frequent.
+  message STREAM_MERGE_OPTIONS {
```

Comment:
To follow the convention, please use CamelCase for the message name, i.e., StreamMergeOptions